Adafruit_USBD_Device.cpp

```diff
@@ -228,6 +228,7 @@ static int8_t utf8Codepoint(const uint8_t *utf8, uint32_t *codepointp)
   else if ((utf8[0] & 0xfe) == 0xfc)
     len = 6;
   else
+    *codepointp = 0
```

Comment:
@henrygab you forgot the bracket, it causes a bigger issue that -1 will always be returned. I think we actually can leave it as it is and have the caller init it rather. Though I am more than OK if you can put a bracket here as well.

```C
static int strcpy_utf16(const char *s, uint16_t *buf, int bufsize)
{
  int i = 0;
  int buflen = 0;

  while (s[i] != 0) {
    uint32_t codepoint = 0; // init here is better
````